tests/queries/0_stateless/02291_filesystem_cache_tests.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+# Tags: long, no-fasttest, no-parallel, no-s3-storage
+
+# set -x
+
+CUR_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CUR_DIR"/../shell_config.sh
+
+TESTS_DIR=$CUR_DIR/filesystem_cache
+TMP_PATH=${CLICKHOUSE_TEST_UNIQUE_NAME}
+
+for NAME in $(find "$TESTS_DIR"/*.sql -print0 | xargs -0 -n 1 basename | LC_ALL=C sort); do
+    TEST_FILE=$TESTS_DIR/$NAME
+    echo "Running tests from: $NAME"
+
+    #for storagePolicy in 's3_lru_cache' 's3_arc_cache'; do
+    for storagePolicy in 's3_lru_cache'; do
+        echo "Using storage policy: $storagePolicy"
```

Comment:
@KinderRiven, can you please continue my script here, need to add remaining tests regarding cache into directory `0_stateless/filesystem_cache`.

Reply:
OK, I don't know if I understand what you mean. I need to put all and cache relevant tests (such as 02226_s3_with_cache.sql, 2240_system_remote_filesystem_cache.sql, 02286_drop_filesystem_cache.sql, ...) to 0_stateless/filesystem. All the test results need to be output to 02291_filesystem_cache_ tests.reference (I mean this file may contain the output results of all cache methods tested)？

Reply:
> @KinderRiven, can you please continue with my script here, need to add remaining tests regarding cache into directory `0_stateless/filesystem_cache`.

OK, I don't know if I understand what you mean. I need to put all and cache relevant tests (such as 02226_s3_with_cache.sql, 2240_system_remote_filesystem_cache.sql, 02286_drop_filesystem_cache.sql, ...) to 0_stateless/filesystem. All the test results need to be output to 02291_filesystem_cache_ tests.reference (I mean this file may contain the output results of all cache methods tested)？

Reply:
yeah, though now I think better to make this `.sh` test per each cache test, then there is not need in a separate directory, we only need to make the underlying test file not ending with `.sql` so it will not be considered as a test file.